zetta_utils/db_annotations/precomp_annotations.py

```diff
@@ -593,16 +622,21 @@ def post_process(self):
           write_annotations(data, False), which writes to only the lowest-level chunks.
         """
         # read data (from lowest level chunks)
+        print("self.read_all")
```

Comment:
Replace print statements with logger calls (see other modules)

Reply:
Oops, the print statements were there for testing and I meant to remove them before committing.  Should be fixed now.